### PR TITLE
[REVIEWS-35] Require approval works now and other fixes

### PR DESCRIPTION
**Pull request by Arturo777**
Require approval works now and other fixes

When you activate the require approval on the app settings, it was not working, but now with this fix, it does.

### CODE REVIEW

File: react/Reviews.tsx [hunk 1]
```diff
@@ -104,7 +105,7 @@ type ReducerActions =
 
 const initialState = {
   sort: 'ReviewDateTime:desc',
-  from: 1,
+  from: 0,
```
Comment (polishq) [inline react/Reviews.tsx:108]: Why was this changed? I think this may cause problems in other parts of the code, for example in the SET_NEXT_PAGE / SET_PREV_PAGE reducer actions

File: dotnet/Services/ProductReviewService.cs [hunk 2]
```diff
@@ -318,6 +318,11 @@ public async Task<ReviewsResponseWrapper> GetReviewsByProductId(string productId
                 searchQuery = $"&_keyword={searchTerm}";
             }
 
+            AppSettings settings = await GetAppSettings();
+            if (settings.RequireApproval)
+            {
+                searchQuery = $"{searchQuery}&approved=true";
```
Comment (polishq) [inline dotnet/Services/ProductReviewService.cs:324]: I think you may need a `_where` argument here, like:
`searchQuery = $"{searchQuery}&_where=approved=true";`

**Review by polishq — CHANGES_REQUESTED** [review]

File: react/Reviews.tsx (hunk 1 above)
Comment (Arturo777) [reply react/Reviews.tsx:108]: it has to be 0 because it means the index of all reviews, if we leave this to 1, the first review (index: 0) always will be missing. and i think the pagination can be fixed by adding one like the last change i made on line 646.

File: dotnet/Services/ProductReviewService.cs (hunk 2 above)
Comment (Arturo777) [reply dotnet/Services/ProductReviewService.cs:324]: this _where is not necessary as i need only a simple one field filter according whit the documentation here: https://developers.vtex.com/vtex-rest-api/reference/search#simple-filter

Comment (polishq) [reply dotnet/Services/ProductReviewService.cs:324]: What if `searchQuery` already has a value? Will it work if you do something like `&_keyword=good&approved=true` ?

File: react/Reviews.tsx (hunk 1 above)
Comment (polishq) [reply react/Reviews.tsx:108]: Ok, yeah, if we change `from` to 0, we also need to change `to` to 9 instead of 10. 
1-10 = 10 reviews
0-10 = 11 reviews 
0-9 = 10 reviews

**Review by polishq — APPROVED** [review]
